Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM swift:5.0.2
+FROM swift:5.1
```

Comment:
is it possible to use 5.1.3 for this too? I thought a bunch of foundation stuff was fixed on linux in releases after 5.1

Reply:
Per this page I believe 5.1 is the same as 5.1.3.

Reply:
https://hub.docker.com/_/swift